PathOfBuildingCommunity/PathOfBuilding
# Predefine build name on character import.

Fixes #8146 .
Fixes #3448 .

### Description of the problem being solved:
Added build name predefinition on character import using this template: "%accountName - %characterName"
It saves predefined name to `main.predefinedBuildName` and propose it to user in save prompt.
### Steps taken to verify a working solution:
- Import Passive Tree or Items
- Click "Save"
- Check edit field.

### Before screenshot:
![image](https://github.com/user-attachments/assets/79ef274e-0ef0-4fe8-b2e5-84151a32cfba)

### After screenshot:
![image](https://github.com/user-attachments/assets/ef6a310c-31f1-4107-9d82-52eaef31ec23)

### Short video:
https://github.com/user-attachments/assets/089c03ce-9f7d-4f42-b13c-67316c2d9951
